src/components/header/header.tsx

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
 import { Sidetittel } from 'nav-frontend-typografi';
 import './header.less';
+import { StatusDropdown } from '../filters/status-dropwdown/status-dropdown';
+import { BrukerFilter } from '../filters/bruker-filter/bruker-filter';
 
 export const Header = () => {
 	return (
 		<div className="header">
-			<Sidetittel>Kvalitetssikring 14a</Sidetittel>
+			<table aria-label="overskrift">
```

Comment:
Jeg tror vi må bruke grid istedenfor table fordi det kan være misvisende for kjermleser brukere

Reply:
Jeg har forsøkt å gjøre det så enkelt som mulig, men syns fortsatt skjermleser er forvirrende.

Reply:
Vi kan også bruke bruke eksisterende <Filters> komponenten i headers.tsx file. 
return (
		<div className="header">
			<Sidetittel className="header__sidetittel">Kvalitetssikring 14a</Sidetittel>
			<Filters />
		</div>
	);

Reply:
return (
		<div className="header">
			<Sidetittel className="header__sidetittel">Kvalitetssikring 14a</Sidetittel>
			<Filters />
		</div>
	);

Reply:
Jeg foreslo å bruke grid istedenfor tabell fordi den er ikke vanlig table som inneholder table header og data.. men her du bruker tabell for å strukturere elementer som kan struktureres med grid.

Reply:
Kan du se på dette igjen?